docker-compose.yml

```diff
@@ -657,6 +657,10 @@ services:
       CONTROL_CENTER_KSQL_WIKIPEDIA_URL: http://ksqldb-server:8088
       # Access ksqlDB server from a browser on your localhost
       CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL: http://localhost:8088
+      # Uncomment following to enable ksqlDB browser access over TLS/HTTPS
```

Comment:
@javabrett this is an excellent addition, thank you.  

1. If we keep this option as a comment, please consider adding verbiage that this replaces the line above `CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL: http://localhost:8088`, to clarify that it is a replacement, not addition.  i.e., user should comment above and uncomment this.  Technically, this is not needed (the 2nd line should have precedence) but it would be helpful to call it out to avoid copy/paste errors and to educate users.

2. Another option we should consider is to go beyond just making a comment, but to add a flag (e.g. something like `ENABLE-KSQLDB-TLS`  when running cp-demo, and created a docker-cmpose overrides file with this parameter.  It's more upfront investment but (a) easier to run and (b) easier to document. I lean towards this option because of its advantages.

Reply:
@ybyzek agreed.  Config for this is now externalised to launch environment via `CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL`, which will default to `http://localhost:8088`.  This comment is left, but we may want to add something in the docs also once reviewed:

```
# To use Control Center and its ksqlDB interface over https, before launching, run in the launch shell:
# export CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL=https://localhost:8089
```